config/packaging/mingw-w64-helics/PKGBUILD

```diff
@@ -3,27 +3,27 @@
 _realname=helics
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
-pkgver=2.0.0
+pkgver=2.3.1
 pkgrel=1
-pkgdesc="A C++ library for co-simulation"
+pkgdesc="Hiearchical Engine for Large-scale Infrastructure Co-Simulation (mingw-w64)"
 arch=('any')
 url="https://github.com/GMLC-TDC/HELICS"
-license=('BSD')
+license=('BSD-3-clause')
+depends=("${MINGW_PACKAGE_PREFIX}-zeromq"
+         "${MINGW_PACKAGE_PREFIX}-python3")
```

Comment:
The python3 package could be added to optdepends instead, so that users using the shared libraries directly can choose if they want to install python.

Reply:
I was debating removing the python stuff completely.  I think it if we could get `pip install` working on MSYS2 then there would be no need to build the python interface as part of this package since that could be loaded if needed through `pip`

Reply:
This PR was more about making some changes in the HELICS build, than about finalizing the msys2 package build script.  That script has to work its way through the MSYS2 checking gauntlet yet, so I expect a number of additional changes before it is finalized.  So I am not concerned if the PKGBUILD is not perfect yet.

Reply:
Is MSYS2 able to use typical Windows dll's?

Reply:
everything on the windows path is in the MSYS2 path, so as long as names don't intersect with the msys2 libraries,  yes

Reply:
I also will not submit the PR to the mingw repos until we get HELICS 2.4 released,  hopefully by then we will have a clear idea of what we can do with pip install.